src/fare-contracts/details/DetailsContent.tsx

```diff
@@ -59,6 +61,14 @@ export const DetailsContent: React.FC<Props> = ({
   const {tariffZones, userProfiles} = useFirestoreConfiguration();
   const {deviceInspectionStatus, barcodeStatus} = useMobileTokenContextState();
 
+  // Checks if the FareContract is purchased by a different ID,
+  // then if yes, return the purchaser ID, otherwise return blank.
+  const purchasedByAccountId =
```

Comment:
it might be renamed by accoundId only because it might be either `purchasedBy` or `customerAccoundId` as I understand customerAccount it the one that got the ticket? and if the `fc.customerAccountId` could be nil if not purchased to someone else the `fc.customerAccountId ?? fc.purchasedBy`, but my logic might be wrong :)

Reply:
`fc.customerAccountId` and `fc.purchasedBy` should not be empty and always contain a value, since the farecontract data have that on the firestore. But yes it might be possible to rename it to a more generic name.